autometa/taxonomy/ncbi.py

```diff
@@ -137,19 +138,17 @@ def lineage(self, taxid, canonical=True):
         taxid : int
             identifer for a taxon in the Taxonomy Database by NCBI
         canonical : bool, optional
-            Returns the names of all the canonical ranks, by default True
+            Returns the names of all the canonical ranks when True and an empty list when False, by default True
 
         Returns
         -------
         ordered list of dicts
             [{'taxid':taxid, 'rank':rank,'name':'name'}, ...]
         """
-        self.is_valid_taxid(taxid)
+        if not self.is_valid_taxid(taxid):
+            return False
```

Comment:
The docstring notes the ReturnTypes as str or None, not bool which is returned if is_valid_taxid is False

Do you want to always return False if the taxid is not valid?